devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -68,11 +68,11 @@ metadata {
 
 		// Yale siren only
 		preferences {
-			input name: "alarmLength", type: "number", title: "Alarm length (1-10 min)", range: "1..10"
+			input name: "alarmLength", type: "number", title: "Alarm length", range: "1..10"
 			// defaultValue: 10
 			input name: "alarmLEDflash", type: "bool", title: "Alarm LED flash"
 			// defaultValue: false
-			input name: "comfortLED", type: "number", title: "Comfort LED (0-25 x 10 sec.)", range: "0..25"
+			input name: "comfortLED", type: "number", title: "Comfort LED (x10 sec.)", range: "0..25"
```

Comment:
What does this do? We're removing units from the title in most other cases so might make sense to do it here too and add a description that both clarifies what this is and indicates the units.

Reply:
I'm honestly not sure what it does.